huawei-wmi.c

```diff
@@ -92,9 +92,8 @@ struct huawei_wmi {
 
 	struct huawei_wmi_debug debug;
 	struct input_dev *idev[2];
-	struct led_classdev cdev;
-	struct led_classdev fn_cdev;
```

Comment:
Could you add the functionality for this one? I.e. a method and the hooks to make it work

Reply:
Displaying the Fn led status was actually my request back then. Although, it proved to differ too much between laptop models and the method to extract the status wasn't particularly reliable (picking a bit out of a byte string from some mildly relevant `\G`-method).

CC @sermart1234

Reply:
Not the LED, actually, the behavior only -- just as on Windows.